src/app/shared/components/description/description.component.html

```diff
@@ -1,18 +1,24 @@
 <div class="container">
-	<div class="text-content">
-		<p #description
-		  [innerHtml]="content"
-		  [ngClass]="{ 'limit-height': isTruncating || heightExceeded, 'full-height': !isTruncating && !isInPopup, 'full-height-popup': !isTruncating && isInPopup }"></p>
-	</div>
+  <div class="text-content">
+    <p #description [innerHtml]="content"
+      [ngClass]="{ 'limit-height': isTruncating || heightExceeded, 'full-height': !isTruncating && !isInPopup, 'full-height-popup': !isTruncating && isInPopup }">
+    </p>
+  </div>
 
-	<ng-container *ngIf="heightExceeded">
-	  <ng-container *ngIf="isTruncating">
-	    <ion-text color="primary" class="button" (click)="isTruncating = !isTruncating">SHOW MORE</ion-text>
-	  </ng-container>
+  <ng-container *ngIf="heightExceeded">
+    <ng-container *ngIf="isTruncating">
+      <div class="button button-container" (click)="isTruncating = !isTruncating">
```

Comment:
Shall we make it a `<ion-button>` instead?

Reply:
ok.